src/entities.ts

```
@@ -71,16 +72,13 @@ customEvents.on('gameLoaded', () => {
-      const WALKING_SPEED = 0.03
-      const SPRINTING_SPEED = 0.18
       const isCrouched = e === bot.entity ? gameAdditionalState.isSneaking : e['crouching']
-      const isWalking = Math.abs(speed.x) > WALKING_SPEED || Math.abs(speed.z) > WALKING_SPEED
-      const isSprinting = Math.abs(speed.x) > SPRINTING_SPEED || Math.abs(speed.z) > SPRINTING_SPEED
 
-      const newAnimation =
-        isCrouched ? (isWalking ? 'crouchWalking' : 'crouch')
-          : isWalking ? (isSprinting ? 'running' : 'walking')
-            : 'idle'
+      const newAnimation = getEntityMovementAnimation({
+        isMounted: e.vehicle != null,
```

ESLint

[error] Use '===' to compare with null.

(no-eq-null)